GNUmakefile

```diff
@@ -203,6 +203,9 @@ check: ## Lint the source code
 	@echo "==> Spell checking website..."
 	@misspell -error -source=text website/pages/
 
+	@echo "==> Checking for breaking changes in protos..."
+	@buf check breaking --against-config buf.yaml --against .git#branch=master
```

Comment:
I'm curious how this works in practice.  Given that latest master is an ever moving target and isn't always the PR base, we may get false positives or build indeterminism. I may suggest using latest stable release, and update it whenever we cut a release.

Few scenarios I'm thinking about:

* if branch is a release backport based on earlier release, if master contains a new field not found in brach, will the linter complain about [a field deletion without reservation](https://docs.buf.build/breaking-checkers/#field_no_delete_unless_number_reserved). The same scenario may happen with actively developed branches that branched off before a new field got merged into master.

* If we iterate on endpoints that hasn't been released, the linter warnings aren't actionable; relatedly, if we accidentally merged a PR with a breaking change, we'll lose the signal unless we go back and examine the branch.  This shouldn't happen, but I've seen cases where failing tests got merged for variety of legitimate reasons or oversight.

Reply:
Glad it looks like it could be helpful - let us know if you have any questions :-)

On your points:

- Checking against a release is likely the best idea overall, we just document as `master` - note that Buf can take inputs from a variety of endpoints, not just your local git branch https://docs.buf.build/inputs
- This `reserved` breaking change checks are some of the more controversial checks, however to make sure breaking change detection is transitive (and to make sure future commits don't include fields that were used in the past but deleted), we chose to enforce this, BUT it's just as easy to disable this using `breaking.except` https://docs.buf.build/breaking-configuration

Reply:
Makes sense - Thank you @bufdev for the insight and feedback!

Reply:
I've updated the `against` to be v1.0.0-beta3; I figure when 1.0.0 is GA we'll change it to that. Seem reasonable?